gneiss/regression/tests/test_ols.py

```diff
@@ -226,6 +240,113 @@ def test_summary_head(self):
             exp = fh.read()
             self.assertEqual(res, exp)
 
+    def test_fit_regularized(self):
+        res = ols(formula="x1 + x2 + x3 + x4",
+                  table=self.y, metadata=self.x, tree=self.t2)
+        res.fit(regularized=True)
+        exp_coefs = pd.DataFrame({
+            'y0': [-2.074816e+09, -2.039271e+08, -1.843143e+08,
+                   2.261170e+09, -7.925699e+06],
+            'y1': [-2.074816e+10, -2.039271e+09, -1.843143e+09,
+                   2.261170e+10, -7.925699e+07]
+            }, index=['Intercept', 'x1', 'x2', 'x3', 'x4']).T
+        # The L1 regularization either has numerical issues
+        # or is random ...
+        res_coefs = res.coefficients()
+        pdt.assert_index_equal(exp_coefs.index, res_coefs.index)
```

Comment:
Minor and I'm unsure if it applies in the pandas testing functions: in general we use `assert*(observed, expected)`, but I'm unsure if the pandas error messages are expecting this convention.